data_collection/gazette/spiders/sp_guaira.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpGuairaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3517406"
+    name = "sp_guaira"
+    allowed_domains = ["guaira.sp.gov.br"]
+    start_urls = ["https://guaira.sp.gov.br/category/diarioficial/"]
+    start_date = datetime.date(2013, 6, 19)
+
+    def extract_edition_number(self, text):
+        edition_number_raw = text.split(" ")[-1]
+        edition_number = ""
+
+        for char in edition_number_raw:
+            if char.isdigit():
+                edition_number += char
+        return edition_number
+
+    def extract_edition_number_extra(self, text):
+        edition_number = ""
```

Comment:
What should be the difference between this and the `extract_edition_number` method?